packages/db-objects-redis/src/lib/objects/objectsInRedisClient.ts

```diff
@@ -1639,46 +1803,45 @@ class ObjectsInRedisClient {
             if (err) {
                 return tools.maybeCallbackWithError(callback, err);
             } else {
-                return this._touch(id, name, options, callback, meta);
+                return this._touch(id, name, callback, meta);
             }
         });
     }
 
-    touchAsync(id, name, options) {
-        return /** @type {Promise<void>} */ (
-            new Promise((resolve, reject) => this.touch(id, name, options, err => (err ? reject(err) : resolve())))
-        );
+    touchAsync(id: string, name: string, options: CallOptions): Promise<void> {
+        return new Promise((resolve, reject) => this.touch(id, name, options, err => (err ? reject(err) : resolve())));
     }
 
-    async _rmHelper(keys, callback) {
-        if (!keys || !keys.length) {
-            return tools.maybeCallback(callback);
+    private async _rmHelper(keys: string[]): Promise<void> {
+        if (!keys.length) {
+            return;
         } else {
             if (!this.client) {
-                return tools.maybeCallbackWithError(callback, ERRORS.ERROR_DB_CLOSED);
+                throw new Error(ERRORS.ERROR_DB_CLOSED);
             }
 
             for (const id of keys) {
-                try {
-                    await this._delBinaryState(id.replace(/\$%\$meta$/, '$%$data'));
-                    await this.client.del(id);
-                } catch (e) {
-                    return tools.maybeCallbackWithRedisError(callback, e);
-                }
+                await this._delBinaryState(id.replace(/\$%\$meta$/, '$%$data'));
```

Comment:
The try catch before was intended ... now it stops on first errornous entry ... befroe it was iterating over the others too

Reply:
Catch lead to return. Thus it is the  same logic as before, only that cb is gone.